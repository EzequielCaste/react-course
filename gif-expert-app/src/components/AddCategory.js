import React, {useState} from 'react'

export const AddCategory = () => {

  const [inputValue, setInputValue] = useState("Hola Mundo");

  const handleInputChange = (event) => {
    setInputValue(event.target.value)
  }

  const handleSumbit = ( e) => {
    e.preventDefault();
    console.log("listo")
  }
  return (
    <form onSubmit={handleSumbit}>      
      <input 
        onChange={handleInputChange}
        type="text"
        value={inputValue}
       />
    </form>
  )
}
