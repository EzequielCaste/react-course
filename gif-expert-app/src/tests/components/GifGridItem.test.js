import React from 'react';
import { shallow } from 'enzyme'
import { GifGridItem } from '../../components/GifGridItem';

describe("Prueba de renderizaci√≥n <GifGridItem />", () => {
  const url = "https://localhost/algo.png"
  const title ="Hola Mundo"

  let wrapper = shallow(<GifGridItem url={url} title={title} />);

  test("debe mostrar <GifGridItem />", () => {
    expect(wrapper).toMatchSnapshot();
  })
})