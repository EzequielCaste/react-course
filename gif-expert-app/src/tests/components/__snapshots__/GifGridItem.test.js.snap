// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de renderizaci√≥n <GifGridItem /> debe mostrar <GifGridItem /> 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Hola Mundo"
    src="https://localhost/algo.png"
  />
  <p>
    Hola Mundo
  </p>
</div>
`;
